### PR TITLE
refactor(editor): unified renderer in separate routes

### PR DESCRIPTION
**Pull request by elbotho**
For https://github.com/serlo/frontend/issues/2933

### Demos/Tools
- [View with mockup data](https://frontend-git-unified-renderer-draft-serlo.vercel.app/static/show-revision)
- [View with mockup data, and revision info](https://frontend-git-unified-renderer-draft-serlo.vercel.app/static/show-revision?revision)
- [Live editor preview](https://frontend-git-unified-renderer-draft-serlo.vercel.app/___editor_preview)  (split panels)
- [/static/[...slug]](https://frontend-git-unified-renderer-draft-serlo.vercel.app/static/1565) to test actual content

### Bundle Size
**+ 25,1 KB** on `static/[...slug]` compared to `[...slug]`

### Overview
image

Right now using the same approach as we landed on for the plugin registry there is a function to register the static renderers before we run the StaticRender. 
This way integrations can choose to use custom static renderers as well, or wrap the default ones with extra code.

We have a default static renderer for each plugin that is inside the plugin folder but only uses code from the `renderer.tsx` and should otherwise be independent from the editor. 

Overall it's very similar to the current frontend-approach, but the code is moved into the editor and runs on serialized editor state directly with no custom types, conversions (no server-side-only steps at all).

### CODE REVIEW

File: src/components/content/static-entity.tsx
Comment (elbotho) [inline src/components/content/static-entity.tsx:1]: (this is just duplicated for the `/static/…` route)

File: src/fetcher/static-create-exercises.ts
Comment (elbotho) [inline src/fetcher/static-create-exercises.ts:1]: (old commented code might still be needed to make exercise folders work)

File: src/fetcher/static-request-page.ts
Comment (elbotho) [inline src/fetcher/static-request-page.ts:1]: (this is just duplicated for the /static/… route with minor changes)

File: src/pages/show-revision.tsx
Comment (elbotho) [inline src/pages/show-revision.tsx:1]: (for demo purposes only)

File: src/pages/static/[...slug].tsx
Comment (elbotho) [inline src/pages/static/[...slug].tsx:1]: (this is just duplicated for the /static/… route with minor changes)

**Review by Entkenntnis — APPROVED** [review]

**Review by hejtful — APPROVED** [review]

**Pull request merged**